charts/stable/kanboard/Chart.yaml

```diff
@@ -8,7 +8,7 @@ keywords:
 - kanboard
 - kanban
 home: https://github.com/truecharts/apps/tree/master/charts/stable/kanboard
-icon: https://truecharts.org/_static/img/appicons/kanboard-icon.png
+icon: https://truecharts.org/_static/img/appicons/kangboard-icon.png
```

Comment:
```suggestion
icon: https://truecharts.org/_static/img/appicons/kanboard-icon.png
```
also rename the icon accordingly!

Reply:
Ok think I fixed kanboard, 

On my side aria looks fixed. Not sure if I am missing something.

Reply:
I'm talking about this lines!
![image](https://user-images.githubusercontent.com/47820033/146692256-c0dffd2d-a041-4580-982b-ee29deee0bb3.png)

At least this is how I see them on master!
https://github.com/truecharts/apps/blob/2761b99c8293b6de93147f03c31d4b29e1dde55e/charts/stable/aria2/Chart.yaml#L8-L9

Reply:
Hmmm I commit'd them. Let me look I prob, messed up something....